### ISSUE TITLE
Support creating Tasks inside Tasks (because it is allowed)

### ISSUE THREAD

**Issue by Robiwahn23** [cschleiden/vsts-quick-decompose#20]
It is possible that a Task can have other Tasks as childs.
So it should be possible from within Decompose to do exactly that :-)

Thanks for this great component!!

**Comment by o-o00o-o** [comment]
I guess the tricky thing here is that there is no validation in VSTS to what can be parent/child to anything. e.g. a task can be a child of an epic/feature etc. 

So the natural conclusion of what you are asking for would be to allow anything into anything. The problem with this is however that VSTS doesn't really want you to do this (even though it is allowed) .

Tasks in tasks don't appear on sprint task boards afaik and pbi's not under features are not visible except on the item cards themselves which is an old and clunky UI (no ordering possible, hard to add new items etc). 

So basically IMO what you are asking for you probably shouldn't do as you are probably making VSTS more difficult to use than it could be.

Just my two pence!
